share/docker/run-tests.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function headline(){
+  echo -e "\e[1;34m#################################################################"
+  echo -e "##### $1 \e[1;37m"
+  echo -e "\e[1;34m#################################################################\e[0m"
+}
+
+set -e
+
+headline "Run Java tests"
+cd /avro/lang/java
+./build.sh test
```

Comment:
What would you say to using a for loop here for all the langs?

Reply:
I've considered it a loop, so let me change it if you prefer a loop 👍